packages/components/src/theme/ComposerTheme/Components/Accordion.ts

```diff
@@ -0,0 +1,18 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Accordion: ComponentStyleConfig = {
+  baseStyle: {
+    panel: {
+      textStyle: 'text-body-regular',
+    },
+  },
+  variants: {
+    highlight: {
+      button: {
+        _expanded: {
+          background: 'background-action-active',
+        },
+      },
+    },
```

Comment:
having a single variant doesn't make a lot of sense to me

Reply:
removed!